malaria24/ona/tests/test_models.py

```diff
@@ -171,6 +171,18 @@ def test_capture_no_ehps(self):
                        'No EHPs found for facility code facility_code.'))
 
     @responses.activate
+    def test_capture_no_ci(self):
+        with LogCapture() as log:
+            ehp = self.mk_ehp()
+            with patch.object(tasks, 'make_pdf') as mock_make_pdf:
+                mock_make_pdf.return_value = 'garbage for testing'
+                case = self.mk_case(facility_code=ehp.facility_code)
```

Comment:
No this shouldn't be the problem. I'm providing the facility code for a facility that doesn't have a case investigator so it should respond with an error.